lib/bucky/core/test_core/test_class_generator.rb

```diff
@@ -15,6 +15,7 @@ module TestClassGeneratorHelper
 
         def add_test_procedure(procedures)
           procedures.each.with_index(1) do |procedure, index|
+            procedure[:proc] ||= ''.dup
```

Comment:
Could you please tell me the reason using `.dup`?

Reply:
Because it will get a string not frozen, when using dup.

```
procedure[:proc] ||= ''"
It will got a string but frozen
```